Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/PowerTransmissions/Axle.cs

```diff
@@ -829,7 +954,69 @@ void StationaryCalculation(float elapsedClockSeconds)
         /// <param name="timeSpan"></param>
         public virtual void Update(float timeSpan)
         {
-            forceToAccelerationFactor = WheelRadiusM * WheelRadiusM / totalInertiaKgm2;
+            // Test to determine whether to use Polach or Pacha adhesion
+                        
+            // Switches between Polach (high performance) adhesion model and Pacha (low performance) adhesion model depending upon the PC performance
+            if(timeSpan < 0.025) // timespan 0.025 = 40 fps screen rate, low timeSpan and high FPS
```

Comment:
We have to add an additional check so that we don't switch between models when simulation is paused (timeSpan=0). Also, when entering dense areas, frame rate will drop and will cause a non-smooth transition between models. That's not good I think.